src/auth/settings.js

```diff
@@ -0,0 +1,7 @@
+export const settings = {
+    authority: process.env.REACT_APP_AUTHORITY,
+    client_id: process.env.REACT_APP_CLIENT_ID,
+    redirect_uri: process.env.REACT_APP_REDIRECT_URL,
```

Comment:
I recommend using "window.location.href" as redirect_uri to return the user to the same page where authentication was requested